.github/workflows/markdown-links-verifier.yml

```diff
@@ -0,0 +1,14 @@
+name: Markdown links verifier
+on: pull_request
+
+jobs:
+  validate_links:
+    name: Markdown links verifier
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout the repository
+      uses: actions/checkout@a81bbbf8298c0fa03ea29cdc473d45769f953675 #@v2
+
+    - name: Validate links
+      uses: Youssef1313/markdown-links-verifier@b3f470e17f1324c6e5401098b910b926515734fb #@v0.0.9
```

Comment:
I don't think this is going to work. The dotnet org has a policy that any actions that are run live in the dotnet organization.

Reply:
@gewarren fwiw, this was accepted in [dotnet/csharpstandard](https://github.com/dotnet/csharpstandard/blob/draft-v6/.github/workflows/markdown-links-verifier.yaml).